phpBB/phpbb/template/twig/tokenparser/defineparser.php

```diff
@@ -23,7 +23,7 @@ class defineparser extends \Twig\TokenParser\AbstractTokenParser
 	*
 	* @return \Twig\Node\Node A Twig\Node instance
 	* @throws \Twig\Error\SyntaxError
-	* @throws \phpbb\template\twig\node\definenode
+	* @returns \phpbb\template\twig\node\definenode
```

Comment:
i think returns is not a valid annotation

Reply:
It is but doesn't really fit in with our other docblocks. Changed it.